clients/apple/Shared/Views/BookView.swift

```diff
@@ -24,8 +26,26 @@ struct BookView: View {
                 .sheet(isPresented: $sheets.renaming) {
                     RenamingSheet()
                 }
+                .toast(isPresenting: Binding(get: { files.successfulAction != nil }, set: { _ in files.successfulAction = nil }), duration: 2, tapToDismiss: true) {
+                    postFileAction()
+                }
     }
-
+    
+    func postFileAction() -> AlertToast {
+        if let action = files.successfulAction {
+            switch action {
+            case FileAction.rename:
+                return AlertToast(type: .regular, title: "File successfully renamed")
```

Comment:
```suggestion
                return AlertToast(type: .regular, title: "File renamed")
```

and so on for the other ones